src/contract/testing/Testing.ts

```diff
@@ -9,12 +14,15 @@ export class Testing {
     await this.arweave.api.get('mine');
   }
 
-  async generateWallet(): Promise<JWKInterface> {
+  async generateWallet(): Promise<Wallet> {
     this.validateEnv();
     const wallet = await this.arweave.wallets.generate();
     await this.addFunds(wallet);
 
-    return wallet;
+    return {
+      jwk: wallet,
```

Comment:
@jakub-wojciechowski additional change in the api that we've talked about.

Reply:
Awesome :)